collections/Don/MS_Don_e_253.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/bodleian/consolidated-tei-schema/master/msdesc.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="https://raw.githubusercontent.com/bodleian/consolidated-tei-schema/master/msdesc.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xml:id="manuscript_9776" xmlns="http://www.tei-c.org/ns/1.0">
+   <teiHeader>
+      <fileDesc>
+         <titleStmt>
+            <title>MS. Don. e. 253</title>
+            <title type="collection">MSS. Don. (Donation)</title>
+            <respStmt>
+               <resp>Cataloguer</resp>
+               <persName>Nigel Palmer</persName>
+            </respStmt>
+         </titleStmt>
+         <publicationStmt>
+            <publisher>
+               <orgName type="department">Special Collections</orgName>
+               <orgName type="unit">Bodleian Libraries</orgName>
+               <orgName type="institution">University of Oxford</orgName>
+               <email>specialcollections.enquiries@bodleian.ox.ac.uk</email>
+            </publisher>
+            <idno type="msID">MS_Don_e_253</idno>
+            <idno type="collection">Don</idno>
+            <idno type="catalogue">Western</idno>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="MS_Don_e_253" xml:lang="en">
+               <msIdentifier>
+                  <country>United Kingdom</country>
+                  <region type="county">Oxfordshire</region>
+                  <settlement>Oxford</settlement>
+                  <institution>University of Oxford</institution>
+                  <repository>Bodleian Library</repository>
+                  <idno type="shelfmark">MS. Don. e. 253</idno>
+                  <altIdentifier type="internal">
+                     <idno type="SCN">Not in SC (late accession)</idno>
+                  </altIdentifier>
+               </msIdentifier>
+               <head>Prayer book; ca. 1500–1520, Bavaria</head>
+               <msContents>
+                  <summary>Collection of German prayers, most likely copied for a lay patron. In all 110 items, some of which forms sets. The following list of contents is selective.</summary>
+                  <textLang mainLang="bar">Central Bavarian, probably from the area between <placeName key="place_7004324">Augsburg</placeName> and <placeName key="place_7003256">Salzburg</placeName>.</textLang>
+                  <msItem n="1">
+                     <locus from="1r" to="1v">(fols. 1r-1v)</locus>
+                     <title>Gebet zur Dreifaltigkeit</title>
```

Comment:
I think these titles (which are editorial) should be translated or at least given in both languages, since a user of our catalogue will be searching from Trinity rather than Driefaltigkeit, for example